contexts/grpc-context/src/main/java/io/spine/logging/context/grpc/GrpcContextDataProvider.java

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2019, The Flogger Authors; 2023, TeamDev. All rights reserved.
+ * Copyright 2025, TeamDev. All rights reserved.
```

Comment:
In other similar places (e.g. [AbstractLogger.java](https://github.com/SpineEventEngine/logging/pull/96/files#diff-91a605b8e52e90b2724de9c8cb5c96ea07584bf02c63bf2ba28fc1352c9e37d9)) the original Flogger copyright remains in-place.

Let's make these changes uniform.